README.md

```diff
@@ -1,150 +1,53 @@
-# 1Password Connect Node SDK
+<!-- Image sourced from https://blog.1password.com/introducing-secrets-automation/ -->
+<img alt="" role="img" src="assets/connect-illustration.svg"/>
 
-[![codecov](https://codecov.io/gh/1Password/connect-sdk-js/branch/main/graph/badge.svg?token=m0PJVEn5m4)](https://codecov.io/gh/1Password/connect-sdk-js)
+<header style="text-align: center;">
+	<h1 style="margin-top: 20px; border-bottom: none;">1Password Connect SDK JS</h1>
+	<p>Access your 1Password items in your JavaScript/TypeScript applications through your self-hosted <a href="https://developer.1password.com/docs/connect">1Password Connect server</a>.</p>
+	<a href="/QUICKSTART.md">
+		<img alt="Get started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37"/>
+	</a>
+</header>
```

Comment:
I may get inspired and use this nice code for my work as well. 😜

Reply:
Please do! I'm updating the template code for all of the other issues as I type this 😅